libdrgn/program.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <sys/statfs.h>
 #include <unistd.h>
+#include <sys/mman.h>
```

Comment:
Alphabetical order for this include, please.

```suggestion
#include <string.h>
#include <sys/mman.h>
#include <sys/statfs.h>
#include <unistd.h>
```